src/prefabs/structures/TimePicker/options/index.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  buttongroup,
+  hideIf,
+  option,
+  property,
+  showIf,
+  text,
+  toggle,
+  variable,
+} from '@betty-blocks/component-sdk';
+import { advanced } from './advanced';
+import { styles } from './styles';
+import { validation } from './validation';
+
+export const options = {
+  actionVariableId: option('ACTION_JS_VARIABLE', {
+    label: 'Action input variable',
+    value: '',
+    configuration: {
+      condition: showIf('property', 'EQ', ''),
+    },
+  }),
+
+  property: property('Property', {
+    value: '',
+    showInReconfigure: true,
+    configuration: {
+      allowedKinds: ['DATE'],
+      disabled: true,
+      allowFormatting: false,
+      condition: hideIf('property', 'EQ', ''),
+      showOnDrop: true,
+    },
+  }),
+
+  label: variable('Label', {
+    value: [''],
+    configuration: { allowFormatting: false, allowPropertyName: true },
+  }),
+  value: variable('Value', {
+    value: [''],
+    configuration: { allowFormatting: false },
+  }),
+
+  type: text('Type', {
+    value: 'datetime',
```

Comment:
dateTime should be time here right?
Also should we even have a type if its only a time picker?